solr/bin/solr

```diff
@@ -2086,14 +2086,14 @@ else
   REMOTE_JMX_OPTS=()
 fi
 
-# Enable java security manager (limiting filesystem access and other things)
-if [ "$SOLR_SECURITY_MANAGER_ENABLED" == "true" ]; then
+# Disable java security manager (allowing filesystem access and other things)
+if [ "${SOLR_SECURITY_MANAGER_ENABLED:-true}" == "true" ]; then
```

Comment:
I think this logic may be reversed. It is true (or unset, in which case it defaults to true), we want to enable the thing. It can be seen from looking quickly at the admin UI at system properties, to determine if you correctly enabled this setting. you will see `java.security.manager` flag set in the system properties part of the admin UI.